### DISCUSSION TITLE
Beatmap's name is called "description"?

### DISCUSSION THREAD

**Discussion by Tenexxt** [ppy/osu#29922] [Q&A] [answered]
When you go to "Details" panel in lazer (the one near the leaderboard options), the difficulty's name is called a description, which should be changed, because its not a description.

I heard that the actual descriptions of maps can't be viewed in lazer because of some issues, so i don't expect for the description to be found on the screen if its not even on the beatmap browser, but this should be renamed _In My Opinion_ (or atleast rename it in the new song select if you don't want to add changes to this one)

![{56FD96E4-8141-4CBF-A355-B1DD40C50B0A}](https://github.com/user-attachments/assets/33e5f6ae-7b58-4a7f-91a5-3f95702a1d1b)
![{9755EFFB-4A2B-4C35-B821-C6E90F99107C}](https://github.com/user-attachments/assets/4c3a71a4-6d89-45eb-a1cc-2ed6aea6f7fa)

**Comment by bdach** [comment, marked as answer]
This design is years past its expiration date. Let's hope we can eventually kill it already.
